### PR TITLE
Mcq generator api

### LINKED ISSUE #24: MCQ Generation
To Do:
- picking an appropriate word in a sentence
- generating options for the choosen word

### PR DESCRIPTION
**Pull request by NAshwinKumar**

### CODE REVIEW

File: src/app/deep-read/setup.sh [hunk 1]
```diff
@@ -50,7 +50,15 @@ fi
...
-python3 -m spacy download en #a language pack required for extracting keywords in MCQs
+
+#Download GloVe model
+GloVeModelFILE=../../../data/training/GloVe/word2vec-glove.6B.300d.txt
+if [ ! -f "$GloVeModelFILE" ]; then
+    echo "GloVe Model not found"
+    gdown https://drive.google.com/uc?id=1ht_zpKv8uXM6LZhUuEpim3cLppLj9GWX -O $GloVeModelFILE
+fi
+
+python3 -m spacy download en #This language pack is required for extracting keywords in MCQs
```
Comment (Acquil) [inline src/app/deep-read/setup.sh]: Can we add this in requirements.txt and install using pip?

File: src/app/deep-read/requirements.txt [hunk 2]
```diff
@@ -9,3 +9,6 @@ flask-restx==0.2.0
 flask-cors==3.0.9
 gdown==3.12.2
 filetype==1.0.7
+fuzzywuzzy==0.18.0
+git+https://github.com/boudinfl/pke.git@6abbcd5d29d14f5151b79a20715ae06dbdd5e4b5
```
Comment (Acquil) [inline src/app/deep-read/requirements.txt:13]: git?

Comment (Acquil) [reply src/app/deep-read/requirements.txt:13]: Think we should add spacy to this list too

Comment (NAshwinKumar) [reply src/app/deep-read/requirements.txt:13]: Yes, pke package is available only in git

File: src/app/deep-read/setup.sh [hunk 3]
```diff
@@ -42,7 +42,7 @@ DIRECTORY=".venv"
 if [ ! -d "$DIRECTORY" ]; then
 # Control will enter here if $DIRECTORY doesn't exist.
     echo "Setting up venv"
-    python3 -m venv ./venv
+    python -m venv ./venv
```
Comment (Acquil) [inline src/app/deep-read/setup.sh]: This should be python3 @NAshwinKumar coz many *nix systems still use python2 as python.

**Review by Acquil — APPROVED** [review]

**Pull request merged**